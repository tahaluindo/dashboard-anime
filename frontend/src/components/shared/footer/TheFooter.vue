<template>
  <footer class="w-full bg-primary">
    <the-footer-container>
      <slot />
      <the-footer-developer-description />
    </the-footer-container>
  </footer>
</template> 

<script lang="ts">
import { defineComponent } from "vue";
import TheFooterDeveloperDescription from "./TheFooterDeveloperDescription.vue";
import TheFooterContainer from "./TheFooterContainer.vue";

export default defineComponent({
  components: {
    TheFooterDeveloperDescription,
    TheFooterContainer,
  },
});
</script>

<style lang="postcss" scoped>
</style>