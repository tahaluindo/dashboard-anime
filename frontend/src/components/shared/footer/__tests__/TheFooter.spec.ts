import { mount } from "@vue/test-utils";
import TheFooter from "../TheFooter.vue";

describe("TheFooter test suit", () => {
  it("should render the default component", async () => {
    const wrapper = mount(TheFooter, {
      shallow: true,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
