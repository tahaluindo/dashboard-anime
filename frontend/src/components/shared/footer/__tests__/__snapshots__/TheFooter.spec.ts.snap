// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter test suit should render the default component 1`] = `
"<footer class=\\"w-full bg-primary\\">
  <the-footer-container-stub></the-footer-container-stub>
</footer>"
`;
